components/dashboards-web-component/src/designer/DashboardDesigner.css

```diff
@@ -96,4 +96,46 @@
 
 .home-page-checkbox {
     padding: 30px 0px;
-}
\ No newline at end of file
+}
+
+.designer-left-menu {
+    padding: 0 !important;
+}
+
+/* dashboard designer golden layout style customizations */
+.dashboard-designer .lm_header.lm_selectable {
+    background: #192830 !important;
+    height: 37px !important;;
+}
```

Comment:
There is an extra semicolon, pls remove.

Reply:
fixed